app/styles/resultat.css

```diff
@@ -23,10 +23,6 @@
     font-size: 80%;
 }
 
-#simulationId {
```

Comment:
It is also used in:
- `app/views/partials/foyer/recap-situation.html`
- `app/views/partials/trace.html`